lib/blather/stanza/iq/query.rb

```diff
@@ -14,7 +14,7 @@ class Query < Iq
     # Overrides the parent method to ensure a query node is created
     #
     # @see Blather::Stanza::Iq.new
-    def self.new(type = nil)
+    def self.new(*)
```

Comment:
To allow creating one with a `to` or `id` set, since the parent constructor already allows those and will get them from the `super` call here. We had a use case for a subclass of this where it was useful to specify the `to`.